packages/aws-cdk/lib/api/resource-import/importer.ts

```diff
@@ -158,9 +220,9 @@ export class ResourceImporter {
         ? ' ✅  %s (no changes)'
         : ' ✅  %s';
 
-      success('\n' + message, this.stack.displayName);
+      await this.ioHost.notify(info(this.action, '\n' + chalk.green(format(message, this.stack.displayName))));
```

Comment:
why did we do away with the `success` helper?

Reply:
Getting rid of all helpers with implicit formatting.